fiduswriter/document/static/js/modules/editor/dialogs/figure.js

```diff
@@ -324,21 +324,27 @@ export class FigureDialog {
             this.aligned = 'center'
         }
 
-        dropdownSelect(
-            this.dialog.dialogEl.querySelector('.figure-width'),
-            {
-                onChange: newValue => {
-                    this.width = newValue
-                    if (this.width == "100") {
-                        alignmentSelector.setValue('center')
-                        alignmentSelector.disable()
-                        this.aligned = 'center'
-                    } else {
-                        alignmentSelector.enable()
+        const figureWidthDOM = this.dialog.dialogEl.querySelector('.figure-width')
+        figureWidthDOM.style.width = "80%"
+        figureWidthDOM.firstElementChild.innerText = `${this.width}%`
```

Comment:
@ArupPatel Can we get the space back so it's `${this.width} %`?

Reply:
done